app/services/events_service.rb

```diff
@@ -0,0 +1,76 @@
+class EventsService
+  attr_accessor :errors, :authentication_token, :params, :current_user
+
+  def initialize(params = {})
+    self.errors = {}
+    self.params = params[:event_params]
+    self.current_user ||= params[:current_user]
+  end
+
+  def create_event
```

Comment:
no need to again add ```_event``` in all methods, since these methods are already inside events_service